Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -62,6 +75,15 @@ public async override Task OnActionExecutionAsync(ActionExecutingContext actionC
             _membershipService = context.HttpContext.RequestServices.GetRequiredService<IMembershipService>();
             _membershipRequestService = context.HttpContext.RequestServices.GetRequiredService<IMembershipRequestService>();
             _newsService = context.HttpContext.RequestServices.GetRequiredService<INewsService>();
+            _CCTContext = context.HttpContext.RequestServices.GetService<CCTContext>();
+            _MyGordonContext = context.HttpContext.RequestServices.GetService<MyGordonContext>();
+
+            // set RecIM services
+            _participantService = new ParticipantService(_CCTContext, _accountService);
+            _matchService = new MatchService(_CCTContext, _accountService);
+            _teamService = new TeamService(_CCTContext, _config, _participantService, _matchService, _accountService);
+            _seriesService = new SeriesService(_CCTContext, _matchService);
+            _activityService = new Services.RecIM.ActivityService(_CCTContext, _seriesService);
```

Comment:
There's a reason we aren't instantiating the services above this section. We want to use the ones in the request that have already been created with dependency injection. Can you please structure these the same way and remove the two contexts?

This is one of the causes of the database context errors that EF Core often gives us.

Reply:
Yep, handled via dependency injection now. Wasn't the one to set it up but it should be ok now.